client/app.jsx

```diff
@@ -27,13 +26,19 @@ export default class App extends React.Component {
 
   renderPage() {
     const { route } = this.state;
+
     if (route.path === '') {
       return <Home />;
     }
 
     if (route.path === 'deck-cards') {
       const deckId = route.params.get('deckId');
-      return <DeckCards deckId={deckId}/>;
+      const tab = route.params.get('tab');
+      const cardIndex = parseInt(route.params.get('cardIndex'));
+      if (!cardIndex) {
+        return <DeckCards deckId={deckId} tab={tab} cardIndex={0} />;
+      }
```

Comment:
You can make this more concise using the `??` nullish coalescing operator.

```js
const cardIndex = parseInt(route.params.get('cardIndex') ?? '0');
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator